fenicsprecice/adapter_core.py

```diff
@@ -128,9 +128,7 @@ def filter_point_sources(point_sources, filter_out):
 
     for point in point_sources.keys():
         # Filter double boundary points to avoid instabilities and create PointSource
-        if filter_out.inside(point, 1):
-            print("Found a double-boundary point at {location}.".format(location=point))
-        else:
+        if not filter_out.inside(point, True):
             filtered_point_sources[point] = point_sources[point]
```

Comment:
Why do you introduce this change? Is this necessary? If not, I would rather leave it as it is.

Reply:
This change is good to have but not really necessary for this specific issue. This was more part of a background clean-up related to `PointSource`s. This will be reverted